homework/01LessonOne.js

```diff
@@ -105,8 +106,9 @@ var notNotTrue;
 //
 // What is the difference between the "and" and the "or" operators? Explain why you would use each of them.
 //
-// Your Answer Goes Here:
-//
+// Your Answer Goes Here: 'And' requires both parts of a problem to be true to
```

Comment:
Awesome! Vocabulary lesson: replace `problem` with `expression` to sound extra fancy.